tools/build_pytorch_libs.py

```diff
@@ -175,6 +175,7 @@ def run_cmake(version,
         USE_CUDA=USE_CUDA,
         USE_DISTRIBUTED=USE_DISTRIBUTED,
         USE_FBGEMM=not (check_env_flag('NO_FBGEMM') or check_negative_env_flag('USE_FBGEMM')),
+        NAMEDTENSOR_ENABLED=(check_env_flag('USE_NAMEDTENSOR') or check_negative_env_flag('NO_NAMEDTENSOR')),
```

Comment:
why do you need to check the env flag here when other examples (e.g. USE_MKLDNN) don't need to?

Reply:
@gchanan the other examples do, somewhere down the line. Here is where the USE_MKLDNN flag is defined, for example: https://github.com/pytorch/pytorch/blob/bc5398451e5de266767120a18f75e08f06d4bf8e/tools/setup_helpers/env.py#L63.

Reply:
ok, I don't know why we don't do things in one place, but if it works, it works!